satnogsclient/scheduler/tasks.py

```diff
@@ -30,9 +30,19 @@ def spawn_observation(*args, **kwargs):
 
 def get_jobs():
     """Query SatNOGS Network API to GET jobs."""
+
+    print 'Try getting Jobs from Network ..'
+
     url = urljoin(settings.NETWORK_API_URL, 'jobs')
     params = {'ground_station': settings.GROUND_STATION_ID}
-    response = requests.get(url, params=params)
+    try:
+        response = requests.get(url, params=params, verify=settings.CA_CERT)
+    except:
+        print 'HTTPs certificate verification failed..(ignore for now)'
+        response = requests.get(url, params=params, verify=False)
```

Comment:
That's what I was also thinking. I didn't meant to hardcode ``verify=False`` in the code rather adding True/False in the settings except of the file path. Anyway, +1 for that.